bolt/change_interval_to_duration.go

```diff
@@ -0,0 +1,127 @@
+package bolt
```

Comment:
I think I would be pro migrations going into their own package but would love to hear what others think.